src/main/scala/scorex/core/network/Handshake.scala

```diff
@@ -19,8 +19,75 @@ case class Handshake(applicationName: String,
   require(Option(protocolVersion).isDefined)
 }
 
+
+class NewHandshakeSerializer(featureSerializers: PeerFeature.Serializers,
+                             maxHandshakeSize: Int) extends scorex.core.newserialization.ScorexSerializer[Handshake] {
+
+  override def parse(r: ScorexReader): Handshake = {
```

Comment:
@VictorMikheev Just a nitpicking here, but in general putting `parse` after `serialize` should be easier to read and follow.

Reply:
fixed